.travis.yml

```diff
@@ -46,7 +46,7 @@ install:
   # and replace the Iris dependency with Iris' underlying dependencies.
   - >
     if [[ "${LIMIT_TO_RELEASED_IRIS}" == false ]]; then
-      IRIS_REF="https://github.com/SciTools/iris/archive/master.zip";
+      IRIS_REF="https://github.com/SciTools/iris/archive/v3.0.0rc0.zip";
```

Comment:
I'm a bit conflicted about this suggestion.
It is a neater way of doing this job -- you don't need code to fetch Iris from two different places in two different ways.  However, it also means the target Iris **must be available in a channel** -- you could no longer use this in a simple way to test against master, or an alternate branch, as we just did.

Personally, I still think that testing against master would be a generally useful thing to do and I'd like to retain that ability for the future.
I will think about how we might do both ...

Did you consider this, and what do you think ?

Reply:
I agree with you. I think we are suffering from crossed purposes:

* **[iris-grib `master`](https://github.com/SciTools/iris-grib/tree/master)**, we have agreed, should test against the latest Iris _release_. I am working on the assumption that the latest _release candidate_ doesn't qualify here.
* **[iris-grib `f__test-iris-master`](https://github.com/SciTools/iris-grib/tree/f__test-iris-master)** was created, as I understand it, to include some features that will only work against yet-to-be released Iris code, so is testing against **[iris `master`](https://github.com/SciTools/iris/tree/master)**.
  This is intended to be functionality that could be re-used in future, so I had imagined that the [altered travis.yml](https://github.com/SciTools/iris-grib/blob/f__test-iris-master/.travis.yml) would eventually make it into **[iris-grib `master`](https://github.com/SciTools/iris-grib/tree/master)**, just with the [`LIMIT_TO_RELEASED_IRIS=false` lines](https://github.com/SciTools/iris-grib/blob/03d60a983f47c0626f466c70bc2ae1e70e87f5d5/.travis.yml#L6-L7) commented out.
* The changes you are proposing in **this PR** are specifically for creating an iris-grib release candidate, to be released alongside [iris v3.0.0rc0](https://github.com/SciTools/iris/releases/tag/v3.0.0rc0). I therefore don't think they fit with the purpose of **[iris-grib `f__test-iris-master`](https://github.com/SciTools/iris-grib/tree/f__test-iris-master)**, as I have understood it, and actually belong in a new release-specific branch.
  My understanding is that release-specific branches can contain release-specific lines not ever intended for the trunk branch, such as the `__version__` string, and that it would therefore be appropriate to have a specific dependency on [iris v3.0.0rc0](https://github.com/SciTools/iris/releases/tag/v3.0.0rc0) here, that would not interfere with the original purpose of **[iris-grib `f__test-iris-master`](https://github.com/SciTools/iris-grib/tree/f__test-iris-master)**.